### PR TITLE
Add documentation for sort_array

### PR DESCRIPTION
**Pull request by xiaoxmeng**
Fix the function list order

### CODE REVIEW

File: velox/docs/functions/array.rst [hunk 1]
```diff
@@ -163,3 +163,14 @@ Array Functions
     If the arguments have an uneven length, missing values are filled with ``NULL`` ::
 
     SELECT zip(ARRAY[1, 2], ARRAY['1b', null, '3b']); -- [ROW(1, '1b'), ROW(2, null), ROW(null, '3b')]
+
+.. function:: array_sort(array(E)) -> array(E)
```
Comment (mbasmanova) [inline velox/docs/functions/array.rst:167]: In the documentation, functions appear in alphabetical order. Would you move this function to preserve that order?

**Review by mbasmanova — COMMENTED** [review]
@xiaoxmeng Thank you for adding documentation for the array_sort function.

PR description repeats PR title unnecessarily. To avoid having duplicate text in the final commit message, please, clear the PR description.

File: velox/docs/functions/array.rst (hunk 1 above)
Comment (xiaoxmeng) [reply velox/docs/functions/array.rst]: Sure. Done

**Review by mbasmanova — APPROVED** [review]
@xiaoxmeng Thank you for fixing the order of the functions.

**Pull request closed without merging**